apps/service_broadcast/lib/broadcast/init.ex

```diff
@@ -5,6 +5,8 @@ defmodule Broadcast.Init do
 
   def on_start(state) do
     Broadcast.Stream.Store.get_all!()
+    |> Enum.reject(&is_nil/1)
+    |> Enum.reject(&Broadcast.Stream.Store.done?(&1))
```

Comment:
would it make more sense to move "done" things to a different collection so they're still able to be referenced but not clogging up the works for regular operations of the app?

Reply:
I'm for that if we get to a point where we see it affect performance.